internal/queue/background.go

```diff
@@ -63,7 +63,7 @@ func (n *noop) Idle(_ time.Duration) error {
 }
 
 // Task is a background task.
-type Task func(ctx context.Context) error
+type Task func(ctx context.Context) (context.Context,error)
```

Comment:
These files are still not formatted. Can you run `make fmt`, please?

Reply:
ran gofmt again, but this time I made sure it acted on all files relevant to this pull request (and also made sure it was the one from go 1.18)